### DISCUSSION TITLE
Track Width ...

### DISCUSSION THREAD

**Discussion by rszemeti** [peterantypas/maiana#34] [Q&A]
I noticed the RF tracks to the RX/TX blocks are quite wide ... certainly a lot wider than 50 Ohms when the top layer is only 0.2mm above the 2nd layer in a typical 7628 style 4 layer stack up ... any particular reason for this, or is the RF ground layer down on Layer 3 and not on layer 2?

**Comment by peterantypas** [comment]
Correct. GND is layer 3.

**Comment by rszemeti** [comment]
Ah, that explains it then ... right, continue as you were :)
